### DISCUSSION TITLE
Blog Posts

### DISCUSSION THREAD

**Discussion by emontes** [blefnk/relivator#21] [Q&A (questions and answers)]
I have not found where or how to save an example blog post in mdx.

I see that in the public/images/ directory there is a blog directory with some images. But I can't find how to add the posts

Should I add them to a particular directory?

**Comment by blefnk** [comment]
@emontes The blog system is still under development. In 1.2.0, it is slightly improved, but still the full functionality will appear in future releases. I am currently running the last tests of 1.2.0 and when it is released, please try playing around with the [Legal](https://github.com/blefnk/relivator/tree/main/src/app/%5Blocale%5D/(legal)) pages ([privacy](https://relivator.bleverse.com/privacy), [terms](https://relivator.bleverse.com/terms)).

**Comment by blefnk** [comment]
✅ [[27.12.2023] v1.2.0 ❄️ Happy New Year 2024 🎁](https://github.com/blefnk/relivator/releases/tag/1.2.0)
